x-pack/plugins/search_indices/public/components/quick_stats/quick_stats.tsx

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import type { Index } from '@kbn/index-management-shared-types';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiI18nNumber,
+  EuiPanel,
+  EuiText,
+  useEuiTheme,
+  EuiButton,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { Mappings } from '../../types';
+import { countVectorBasedTypesFromMappings } from './mappings_convertor';
+import { QuickStat } from './quick_stat';
+
+export interface QuickStatsProps {
+  index: Index;
+  mappings: Mappings;
+}
+
+export const SetupAISearchButton: React.FC = () => {
+  return (
+    <EuiPanel hasBorder={false} hasShadow={false} color="transparent">
+      <EuiFlexGroup gutterSize="s" direction="column" alignItems="center">
+        <EuiFlexItem grow={false}>
+          <EuiText>
+            <h6>
+              {i18n.translate('xpack.searchIndices.quickStats.setup_ai_search_description', {
+                defaultMessage: 'Use Elastic to build powerful AI Search',
+              })}
+            </h6>
+          </EuiText>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiButton
+            href="https://www.elastic.co/guide/en/elasticsearch/reference/current/semantic-search.html"
```

Comment:
should this be in a doc link?

Reply:
Yes thanks! Now we have docLinks, can add this.